lessons/02-javascript/README.md

```diff
@@ -152,6 +157,8 @@ for (var key in object) {
 }
 ```
 
+Note that the order of iteration is not guarenteed with this syntax and later, if you use libraries which modify the prototype (more on this later) of base types, can have strange behavior.
```

Comment:
Good point, might as well show them the better syntax from the start. I don't think I used forEach() until the hamburger app.